### PR TITLE
chore(macros/SectionOnPage): add l10n-zh translation

### PR DESCRIPTION
**Pull request by yin1999**
## Summary

add l10n-zh translation for `{{SectionOnPage}}` macro.

### After

In `http://localhost:5042/zh-CN/docs/Web/API/Clipboard/write`

image

---

## How did you test this change?

Run `yarn dev`.

### CODE REVIEW

**Comment by yin1999** [comment]
Poke @irvin for a review (`zh-TW` translation)

**Review by caugner — APPROVED** [review]

**Review by caugner — COMMENTED** [review]
Will merge once @irvin has approved.

**Review by irvin — APPROVED** [review]

**Pull request merged**
